clippy_lints/src/disallowed_macros.rs

```diff
@@ -71,11 +71,11 @@ pub struct DisallowedMacros {
 
     // When a macro is disallowed in an early pass, it's stored
     // and emitted during the late pass. This happens for attributes.
-    earlies: AttrStorage,
+    earliest: AttrStorage,
```

Comment:
That's not a typo. This stores the "early macros", or "early-s" (here written as "earlies").